# Service to Set Joints


---
int64 ret
string msg
